src/test/java/com/scottlogic/SortingTests/AuthorSortingTests.java

```diff
@@ -0,0 +1,72 @@
+package com.scottlogic.SortingTests;
+
+import com.scottlogic.SortContext;
+import com.scottlogic.SortOrder;
+import com.scottlogic.SortingTests.utils.UserPostSortData;
+import com.scottlogic.UserPost;
+import com.scottlogic.sort.SortAuthor;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public class AuthorSortingTests {
+
+    SortContext sortAuthor = new SortContext(new SortAuthor());
+
+    @Test
+    @DisplayName("Ascending functionality")
+    void sortAuthorAscending() {
+        List<UserPost> postsToSort = UserPostSortData.getUnsortedList();
+        List<UserPost> expectedAuthorSort = UserPostSortData.getSortedAuthorAscendingList();
+        Assertions.assertEquals(expectedAuthorSort, sortAuthor.sortPosts(postsToSort, SortOrder.ASC));
+    }
+
+    @Test
+    @DisplayName("Descending functionality")
+    void sortAuthorDescending() {
+        List<UserPost> postsToSort = UserPostSortData.getUnsortedList();
+        List<UserPost> expectedAuthorSort = UserPostSortData.getSortedAuthorDescendingList();
+        Assertions.assertEquals(expectedAuthorSort, sortAuthor.sortPosts(postsToSort, SortOrder.DESC));
+    }
+
+    @Nested
+    @DisplayName("Empty input")
+    class sortAuthor_EmptyInput {
+        @Test
+        @DisplayName("ascending sort")
+        void sortAscending_emptyInput() {
+            List<UserPost> emptyList = Arrays.asList();
+            Assertions.assertEquals(emptyList, sortAuthor.sortPosts(emptyList, SortOrder.ASC));
+        }
+
+        @Test
+        @DisplayName("descending sort")
+        void sortDescending_emptyInput() {
+            List<UserPost> emptyList = Arrays.asList();
+            Assertions.assertEquals(emptyList, sortAuthor.sortPosts(emptyList, SortOrder.DESC));
+        }
+    }
+
+    @Nested
+    @DisplayName("Null input")
```

Comment:
Inconsistent naming on display names.
ascending sort / Null input / descending sort / Empty input

Reply:
Was going for '**Null input**' as kind of a heading, then the lowercase '_a_scending input' for a sub-heading - what do you think? or just keep capitals consistent?

Reply:
What does it look like when logged out?

Reply:
![image](https://user-images.githubusercontent.com/93915369/145791488-fe0a9fad-96b7-4c56-a890-7ae823004448.png)

Reply:
Tried using <img> tag to resize but didn't work sorry